scripts/util/gh.py

```diff
@@ -0,0 +1,13 @@
+REPO_RUNNER_LABEL_MAP = {
+    "amd64": "X64",
```

Comment:
btw, the self-hosted runners team did change that and now also appends the `amd64` label to those runners.
There is still an inconsistency thought ARM64 and amd64 (1x uppercase, 1x lowercase), so I guess we should just be explicit here and keep using the map.